modules/core/src/main/java/org/apache/ignite/internal/processors/configuration/distributed/DistributedConfigurationProcessor.java

```diff
@@ -236,9 +236,8 @@ private void doClusterWideUpdate(DistributedChangeableProperty prop) {
                 return distributedMetastorage.writeAsync(toMetaStorageKey(key), newValue);
             }
 
-            @Override
-            public GridFutureAdapter<?> casUpdate(String key, Serializable expectedValue, Serializable newValue)
-                throws IgniteCheckedException {
+            @Override public GridFutureAdapter<?> casUpdate(String key, Serializable expectedValue,
```

Comment:
According to codestyle, each arguments must be on a new line.

Reply:
fixed